src/modules/home/communityBenefits/CommunityBenefitCard.tsx

```diff
@@ -0,0 +1,21 @@
+import Image from "next/image";
+
+type CommunityBenefitsCardProps = {
```

Comment:
Not a big one, but if component name is `CommunityBenefitCard` then why it is plural in type name ?

Reply:
Also you have defined the type already on helper, why to repeat same type here again ?